src/ray/core_worker/core_worker.cc

```diff
@@ -3,7 +3,7 @@
 
 namespace ray {
 
-CoreWorker::CoreWorker(const enum WorkerType worker_type, const ::Language language,
+CoreWorker::CoreWorker(const enum WorkerType worker_type, const enum Language language,
```

Comment:
no need `enum` here I think

Reply:
needed because there's a method `Language()` in `CoreWorker`.

Reply:
We should resolve the duplicated name issue by namespace rather than enum, right?

Reply:
I tried googling this, but couldn't find why using namespace is more preferable in this case.